src/Cuberite.vcxproj.user

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure if VS will accept it this way. Did you check?
(The file is only used to provide defaults for when the project is first ever opened)

Reply:
Changed settings and the BOM is not readded.

Reply:
As I said, this file is only ever used for the defaults. Your settings are then saved to another file.

Reply:
I changed the working directory for x64 release / debug builds and VS changed this file. This are the changes:
```
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
    <LocalDebuggerWorkingDirectory>..\Server</LocalDebuggerWorkingDirectory>
    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
  </PropertyGroup>
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
    <LocalDebuggerWorkingDirectory>..\Server</LocalDebuggerWorkingDirectory>
    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
  </PropertyGroup>
```

Reply:
That is somewhat unexpected, I suppose VS behavior with these files changed. Feel free to add it.

Reply:
Although I actually wanted to make CMake write this file, since the build can be started out-of-source, which would not use this file.